lib/pages/faq/service/question_provider.dart

```diff
@@ -1,18 +1,25 @@
+import 'package:acs_upb_mobile/authentication/model/user.dart';
+import 'package:acs_upb_mobile/authentication/service/auth_provider.dart';
 import 'package:acs_upb_mobile/generated/l10n.dart';
 import 'package:acs_upb_mobile/pages/faq/model/question.dart';
 import 'package:acs_upb_mobile/widgets/toast.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
 
 class QuestionProvider with ChangeNotifier {
-  Future<List<Question>> fetchQuestions({int limit}) async {
+  Future<List<Question>> fetchQuestions(
```

Comment:
I agree it might be out of the scope of this PR. 
When I talked to @AdrianMargineanu , he mentioned integrating the source selection functionality in those parts altogether: FAQ, upcoming events, favorite websites
Maybe should have been thought to be a more gradual integration, not in the same PR, but at least on the FAQ side, I tested it and it all worked fine

Reply:
Renamed this